package.json

```diff
@@ -51,15 +48,15 @@
     "query-string": "4.1.0",
     "react": "15.4.1",
     "react-ace": "3.7.0",
-    "react-addons-css-transition-group": "15.4.1",
     "react-addons-pure-render-mixin": "15.4.1",
     "react-addons-test-utils": "15.4.1",
     "react-dom": "15.4.1",
     "react-gemini-scrollbar": "2.1.5",
     "react-intl": "2.2.3",
     "react-redux": "4.4.6",
     "react-router": "2.8.1",
-    "reactjs-components": "0.16.0-beta.23",
+    "react-transition-group": "1.2.1",
```

Comment:
@bstavroulakis wondering why we need this adjustment in this PR? It seems unrelated to the scope of deleting status.

Please squash https://github.com/dcos/dcos-ui/pull/2440/commits/23710979e0f1a61423dd850017099ee942a21095 with https://github.com/dcos/dcos-ui/pull/2440/commits/89a306d59dffd44245d225e3e17b498a3737482e

so we can maintain a clean commit history. The final commit message should then be edited as I don't think the scope of `feat(ServiceDeleteActions)` is correct.  This commit should also be extracted and put in a separate PR so we can test the scope of CSS transitions separately from the scope of service delete status.

Please also squash https://github.com/dcos/dcos-ui/pull/2440/commits/89a306d59dffd44245d225e3e17b498a3737482e#diff-2f23d4286259700270abc15fd4492804 into https://github.com/dcos/dcos-ui/pull/2440/commits/9d429ee9adae684b11cc9aba7035153a140d505d so we don't have commit's with rebase conflict markup.